experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -420,18 +422,17 @@ export class FetchInstrumentation extends InstrumentationBase<
   private _prepareSpanData(spanUrl: string): SpanData {
     const startTime = core.hrTime();
     const entries: PerformanceResourceTiming[] = [];
-    if (typeof window.PerformanceObserver === 'undefined') {
+    if (typeof PerformanceObserver === 'undefined') {
```

Comment:
```suggestion
    if (PerformanceObserver == null) {
```

Reply:
Oh right, you reminded me with the new style guide! :)